WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -157,7 +160,14 @@ extension ImageAndTitleAndTextTableViewCell {
         descriptionLabel.numberOfLines = viewModel.numberOfLinesForText
         contentImageView.image = viewModel.image
         contentImageStackView.isHidden = viewModel.image == nil
-        accessoryType = viewModel.isActionable ? .disclosureIndicator : (viewModel.isSelected ? .checkmark : .none)
+        accessoryType = viewModel.showsDisclosureIndicator ? .disclosureIndicator : .none
+        if viewModel.isActionable && viewModel.showsDisclosureIndicator == false {
+            accessoryType = .none
+        } else if viewModel.isActionable == false && viewModel.showsDisclosureIndicator && viewModel.isSelected {
+            accessoryType = .checkmark
+        } else if viewModel.isActionable == false && viewModel.showsDisclosureIndicator && viewModel.isSelected == false {
+            accessoryType = .none
+        }
```

Comment:
Is there a cleaner way to write this bit? I found myself being very specific with the cases because we're using all of them across the app in a way that (seems to be) inconsistent manner, for example, we hide the disclosure indicator in Order Creation and show it on "Add Files" when both are actionable and start a new flow, so showing and indicator would be expected (or hide it in both).

Reply:
From what I gathered, a cell's `isActionable` is only false when it contains some UI component that can be interacted with (e.g a switch) - in this case we want to not highlight the cell when the cell is touched. In other words, `isActionable` should [only concern the selection state](https://github.com/woocommerce/woocommerce-ios/blob/8ea8883aa3f97f422ebb1218e3b0fa6f19e44320/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/Cells/ImageAndTitleAndTextTableViewCell.swift#L171) of the cell and not the `accessoryType`, which is the solution we want to aim.

My suggestion is to omit `isActionable` from all these checks:

```suggestion
        if viewModel.showsDisclosureIndicator {
            accessoryType = disclosureIndicator
        } else if viewModel.isSelected {
            accessoryType = .checkmark
        } else {
            accessoryType = .none
        }
```

Reply:
Thanks, this looks much more cleaner and understandable now. I was definitely having a misunderstanding on the difference between .isActionable, .isSelected, and where to use them.